addresses.json

```diff
@@ -1,213 +1,231 @@
-    "bullaInstantPaymentAddress": "0xFcA0E74af938919E43541D40330212324C8aF27F",
-    "batchCreate": {
-      "address": "0x1c534661326b41c8b8aab5631ECED6D9755ff192",
-      "maxClaims": 40
+    "42220": {
+        "name": "Celo Mainnet",
+        "deployedOnBlock": 9469593,
+        "bullaManagerAddress": "0x68da2c92b60337aA55BAcF0d7e61126eDb535752",
+        "bullaBankerAddress": "0xd2A779d4B8d2E4E7faFC0DA6F9d712fE19464D5B",
+        "bullaClaimERC721Address": "0x4Ba1453df995F4AAe2ef610A9c06fEecF0Fe2581",
+        "bullaInstantPaymentAddress": "0x48D283521Ff91a1e83A16a7138B549C34BeDD44c",
+        "batchCreate": {
+            "address": "0xe14e624b29bcda2ec409bbbf97037fede3803797",
+            "maxClaims": 40
+        },
+        "moduleFactoryAddress": "0xD5585F346414d57B04A9798F246C7DBfDf64fC35",
+        "masterCopyAddress": "0x25e6F45D4D8db871bb5D7F50D42EC7BcbDF32Fb3"
     },
-    "moduleFactoryAddress": "0x101452DEF98B40A64d28A964Df1e35170aD5c647",
-    "bullaModuleMasterCopyAddress": "0xF9C5C979c4d80ADf5Da07e943658617d0110cC92"
-  },
-  "84531": {
-    "name": "base_goerli",
-    "deployedOnBlock": 3375601,
-    "bullaManagerAddress": "0x1b4DB52FD952F70d3D28bfbd406dB71940eD8cA9",
-    "bullaBankerAddress": "0x873C25e47f3C5e4bC524771DFed53B5B36ad5eA2",
-    "bullaClaimERC721Address": "0x127948A4286A67A0A5Cb56a2D0d54881077A4889",
-    "batchCreate": {
-      "address": "0x6811De39DC03245A15D54e2bc615821f9997bbC4",
-      "maxClaims": "255"
+    "43114": {
+        "name": "Avalanche Mainnet",
+        "deployedOnBlock": 5968486,
+        "bullaManagerAddress": "0xAA6E5B4E34f3C3BA4D90694909dca7DDdf058079",
+        "bullaBankerAddress": "0x4Ba1453df995F4AAe2ef610A9c06fEecF0Fe2581",
+        "bullaClaimERC721Address": "0x68da2c92b60337aA55BAcF0d7e61126eDb535752",
+        "bullaInstantPaymentAddress": "0xFcA0E74af938919E43541D40330212324C8aF27F",
+        "batchCreate": {
+            "address": "0x1c534661326b41c8b8aab5631ECED6D9755ff192",
+            "maxClaims": 40
+        },
+        "moduleFactoryAddress": "0x101452DEF98B40A64d28A964Df1e35170aD5c647",
+        "bullaModuleMasterCopyAddress": "0xF9C5C979c4d80ADf5Da07e943658617d0110cC92"
     },
-    "bullaInstantPaymentAddress": "0xec6013D62Af8dfB65B8248204Dd1913d2f1F0181",
-    "bullaFinanceAddress": "0x26719d2A1073291559A9F5465Fafe73972B31b1f",
-    "frendLendAddress": "0xd3A33aE646701507eB043e2DB16f8C1428241F53"
-  },
-  "1313161554": {
-    "name": "Aurora Mainnet",
-    "deployedOnBlock": 59115995,
-    "bullaBankerAddress": "0xeB0f09EEF3DCc3f35f605dAefa474e6caab96CD6",
-    "bullaManagerAddress": "0xE14E624b29BcDa2ec409BBBf97037fEDe3803797",
-    "bullaClaimERC721Address": "0x1c534661326b41c8b8aab5631ECED6D9755ff192",
-    "bullaInstantPaymentAddress": "0x1b4DB52FD952F70d3D28bfbd406dB71940eD8cA9",
-    "batchCreate": {
-      "address": "0x0af8C15D19058892cDEA66C8C74B7D7bB696FaD5",
-      "maxClaims": 40
+    "84531": {
```

Comment:
Changing all this formatting is going to make the merge a total nightmare

Reply:
If I remove whitespace I don't see it as a change. How should I handle this?